packages/plexus/src/DirectedGraph/builtins/PureEdges.tsx

```diff
@@ -31,6 +31,8 @@ export default class PureEdges extends React.PureComponent<TProps> {
     return layoutEdges.map(edge => (
       <EdgePath
         key={`${edge.edge.from}\v${edge.edge.to}`}
+        id={`${edge.edge.from}-${edge.edge.to}`}
```

Comment:
can you add a comment to this class? What is it used for, what are "pure edges"?